### PR TITLE
add: new quotes from The Knight in Rusty Armour

### PR DESCRIPTION
**Pull request by D3press3dd**
### Description
add: new quotes from The Knight in Rusty Armour

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4798963151

File: frontend/static/quotes/spanish.json [hunk 1]
```diff
@@ -816,6 +816,30 @@
...
       "id": 135
+    },
+    {
+      "text": "El miedo a su propia sombra lo tenía atrapado y le impedía ver la luz del sol.",
+      "source": "Robert Fisher - El caballero de la armadura oxidada",
+      "length": 78,
+      "id": 136
+    },
+    {
+      "text": "El miedo al rechazo, al abandono, a no ser suficientemente bueno o querido, nos lleva a creer que necesitamos armaduras para protegernos, cuando en realidad lo que necesitamos es quitárnoslas para mostrar nuestra verdadera esencia y conectarnos auténticamente con los demás",
```
Comment (Miodec) [inline frontend/static/quotes/spanish.json:827]: Missing period at the end

File: frontend/static/quotes/spanish.json [hunk 2]
```diff
@@ -816,6 +816,30 @@
...
+      "source": "Robert Fisher - El caballero de la armadura oxidada",
+      "length": 273,
+      "id": 137
+    },
+    {
+      "text": "La vida es como un espejo: si sonríes, el espejo te devuelve la sonrisa; pero si muestras una cara larga, recibes lo mismo a cambio, por lo que siempre es mejor sonreír a la vida y mantener el corazón abierto a las posibilidades que nos ofrece cada día",
```
Comment (Miodec) [inline frontend/static/quotes/spanish.json:833]: Period

File: frontend/static/quotes/spanish.json [hunk 3]
```diff
@@ -816,6 +816,30 @@
...
+      "source": "Robert Fisher - El caballero de la armadura oxidada",
+      "length": 252,
+      "id": 138
+    },
+    {
+      "text": "A veces, para encontrar el camino hacia la luz, es necesario pasar por la oscuridad, enfrentar nuestros miedos, sanar nuestras heridas y aprender las lecciones que la vida nos está enseñando para poder renacer con una armadura nueva y brillante, llena de amor y sabiduría",
```
Comment (Miodec) [inline frontend/static/quotes/spanish.json:839]: Period

**Pull request closed without merging**